airqo-platform/AirQo-frontend
# Add logo to the platform

- What does this PR do?
This PR add AirQo, KCCA, Mak logos. It also updates to Toolbar background colour to correspond to AirQo colour code.

- Which JIRA board item is attached to this PR?
[This one
](https://airqoteam.atlassian.net/browse/PLAT-90)
- How can I test it out?
cd AirQo-frontend/netmanager
npm install

> for MAC users

```
npm run dev-mac
npm run stage-mac
npm run prod-mac
```

> for Windows users

```
npm run dev-pc
npm run stage-pc
npm run prod-pc
```

> Alternatively:
> You can test out using Docker:

```
npm run build
docker build -t us.gcr.io/airqo-frontend/airqo-stage-netmanager-frontend .
docker run -d -p 8888:80 us.gcr.io/airqo-frontend/airqo-stage-netmanager-frontend
```

- Under what environment should I test it out?
node v12.18.1

![Screenshot 2020-09-17 at 18 26 20](https://user-images.githubusercontent.com/15224992/93494220-73c2d800-f915-11ea-8e68-125422f77190.png)
